### PR TITLE
🎨 [Frontend] TIP: Update Texts and Quick Start visuals

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

Curated texts.

- [x] Disclaimer text
- [x] New plans
- [x] Buy Credits Guided tour
- [x] Quick Start:
  - [x] Visuals link to manual content
  - [x] Open visual in new tab on click
  - [x] Fit visuals to window width

Disclaimer text:
image

New plans
image

Buy Credits Guided tour
BuyCredits

QuickStart:
QuickStart

## Related issue/s

closes https://github.com/ITISFoundation/osparc-issues/issues/1571

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]

**Comment by odeimaiz** [comment]
> new plans?

Last steps before crossing the pond!

**Review by elisabettai — APPROVED** [review]
Super-nice.

Minor on the "Buy Credits" guided tour:
-  it might not be very clear to new users what is credit indicator. Can you highlight it in the UI when it is selected in the guided tour? Maybe also renaming to "Credits Icon" might helpt.
- the text of the guided tour might need to be updated to reflect the labels on the buttons (e.g. "Auto-recharge" instead of "Auto top-up", but also others).

**Review by GitHK — APPROVED** [review]

File: services/static-webserver/client/source/resource/osparc/tours/tis_tours.json [hunk 1]
```diff
@@ -98,46 +98,46 @@
...
-      "text": "This is the list of Credits Accounts you have access to. You have your own Account, but you could also use Credit Accounts that were shared with you.",
+      "title": "Credit Accounts",
+      "text": "This lists the Credit Accounts you have access to. Apart from your own Account, the list also includes Credit Accounts that were shared with you.",
       "placement": "bottom"
     }, {
       "anchorEl": "osparc-test-id=personalWalletsList",
-      "title": "Top-up Credits",
-      "text": "There are two ways to charge credits into your Credit Account: you can set up an Auto-recharge process or go for a One time payment.",
+      "title": "Buy Credits",
+      "text": "There are two ways to add Credits to your Credit Account: you can set up an auto-top-up procedure or make a one-time payment.",
```
Comment (jsaq007) [inline services/static-webserver/client/source/resource/osparc/tours/tis_tours.json:123]: _There are two ways to add Credits to your Credit Account: you can set up an auto-top-up procedure or make a one-time payment._

**There are two ways to add Credits to your account: you can set up and select the auto-top-up option or by making one-time payments.**

File: services/static-webserver/client/source/resource/osparc/tours/tis_tours.json [hunk 2]
```diff
@@ -98,46 +98,46 @@
...
+      "title": "Auto-Recharge",
+      "text": "Keep your balance running smoothly by automatically setting your Credits to be topped up when it runs low. Before the automated top-up function can be activated, you must add a payment method.",
       "placement": "bottom"
     }, {
-      "anchorEl": "osparc-test-id=autorechargeBtn",
-      "title": "Auto-recharge",
-      "text": "Keep your balance running smoothly by automatically setting your credits to be recharged when it runs low.<br>Before the auto-recharge function can be activated you need to add your first payment method.",
+      "anchorEl": "osparc-test-id=buyCreditsBtn",
+      "title": "One-Time payment",
+      "text": "You can top up your Credits with a one-off, non-recurring payment. After the payment is made, the new Credits will be added to your Credit Account. The payment method can be saved for future top ups.",
```
Comment (jsaq007) [inline services/static-webserver/client/source/resource/osparc/tours/tis_tours.json:133]: We should probably remain consistent with our terminology; is it one-off payment or one-time payment?

**Review by jsaq007 — APPROVED** [review]
A couple consistency things I spotted. Not a show stopper but nice have. The other thought I had was the concatenation of strings could also be more consistent. I guess this is my doing coming from react which doesn't like the var + 'string' way but prefers the **`** (backticks) which if memory serves is the new norm.

File: services/static-webserver/client/source/resource/osparc/tours/tis_tours.json (hunk 1 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/resource/osparc/tours/tis_tours.json:123]: https://github.com/ITISFoundation/osparc-simcore/pull/6112#issuecomment-2252735810

**Pull request merged**